lesson-06/hw6/example-05.js

```diff
@@ -0,0 +1,8 @@
+import users from './users.js';
+
+const getUserWithEmail = (users, email) => {
```

Comment:
Якщо тіло функції досить просте, то можна одразу повертати значення 
```suggestion
const getUserWithEmail = (users, email) => users.find(user => user.email === email);
```